### ISSUE TITLE
error after changing the Network from TFlearn code to TF slim code

### ISSUE THREAD

**Issue by hjl240** [pppoe/tensorflow-sphereface-asoftmax#5]
hi, I change the Network from TFlearn code to TF slim code, the tf-slim code as follow
`def prelu(_x, scope=None):
    """parametric ReLU activation"""
    with tf.variable_scope(name_or_scope=scope, default_name="prelu"):
        _alpha = tf.get_variable("prelu", shape=_x.get_shape()[-1],
                                 dtype=_x.dtype, initializer=tf.constant_initializer(0.1))
        return tf.maximum(0.0, _x) + _alpha * tf.minimum(0.0, _x)

def Network(data_input, training = True):
    x = slim.conv2d(data_input, 32, [3,3], stride = 1, activation_fn=prelu)
    x = slim.conv2d(x, 32, [3, 3], stride = 2, activation_fn=prelu)
    x = slim.conv2d(x, 64, [3, 3], activation_fn=prelu)
    x = slim.conv2d(x, 64, [3, 3], stride = 2, activation_fn=prelu)
    x = slim.conv2d(x, 128, [3, 3], activation_fn=prelu)
    x = slim.conv2d(x, 128, [3, 3], stride = 2, activation_fn=prelu)
    x = slim.flatten(x)
    feat = slim.fully_connected(x, 2)
    return feat
`
when I train the network, the loss does not decline, how can i solve this problem?Thanks

_0 2.3063145
100 2.3025854
200 2.3025854
300 2.3025854
400 2.3025854
500 2.3025854
600 2.3025854_

------------------
sloved! just change "feat = slim.fully_connected(x, 2)" to "feat = slim.fully_connected(x, 2,activation_fn=prelu)"
